homeassistant/components/cookidoo/manifest.json

```
@@ -6,6 +6,7 @@
   "documentation": "https://www.home-assistant.io/integrations/cookidoo",
   "integration_type": "service",
   "iot_class": "cloud_polling",
+  "loggers": ["cookidoo", "cookidoo-api"],
```

```suggestion
  "loggers": ["cookidoo-api"],
```
Most likely you dont need the other one

---

Not sure, as debug logs from within the integration were also not visible with debug mode enabled. Only when switching to info, they were displayed. @joostlek

---

Hmm, something was bizarre, now debug logs from within the integration appear also without this setting.